### DISCUSSION TITLE
Transparent background of png image in header is rendered with a white background

### DISCUSSION THREAD

**Discussion by jweickm** [marp-team/marp#345] [Q&A] [answered]
I am looking to include a png logo in the header (or footer) that has a transparent background. However, it is rendered with a white background, making it unusable in combination with slides that use `[bg](some-background-image.jpg)`. Am I missing something or is it simply not possible to include logos in the header/footer in this way?

Edit: added image for illustration
![logo](https://user-images.githubusercontent.com/57487288/188950902-95c51e04-3eb7-4d77-9b26-77d565e215ea.jpg)

This is the slide code:

```
---
marp: true
paginate: false
theme: default
---

<!-- header: ![h:3em](./logos/logo.png) -->

![bg](pictures/berlin.jpg)
```

**Comment by yhatt** [comment, marked as answer]
Marp's default theme is based on [GitHub's Markdown style](https://github.com/sindresorhus/github-markdown-css), and [its images always have a constant background color](https://github.com/sindresorhus/github-markdown-css/blob/ec90c8c72e77ca5f0a6b048514a11525808d00bd/github-markdown.css#L206).

For getting backward compatibility of visibility with the slide made by [classic Marp app](https://github.com/yhatt/marp), we are keeping this implicit background color over 6 years (when using regular color scheme). If you want, [you can tweak the style of `<img>` elements](https://marpit.marp.app/theme-css?id=tweak-style-through-markdown) to make image's background transparency.

```html
<style>
  img {
    background-color: transparent;
  }
</style>
```

**Reply by jweickm** [reply]
いつもお世話になっております。
Thank you for your incredibly fast and comprehensive reply. That helps me to adjust the theme. Really have to dive deeper into CSS customization.
Thanks again for making this amazing framework. Can't imagine creating presentations with the likes of Powerpoint and co. anymore.
